### PR TITLE
fix(runtime): Fixed the error of packing and running the Times.

### PR DESCRIPTION
**Pull request by zzcr**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/ui-vue/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: internals/cli/src/commands/build/build-ui.ts [hunk 1]
```diff
@@ -93,57 +93,58 @@ export const getBaseConfig = ({ vueVersion, dtsInclude, dts, buildTarget, themeV
...
-              dependencies[key] = value
-            }
-          })
-
-          if (filePath.includes('vue-common') && vueVersion === '2') {
-            dependencies['@vue/composition-api'] = '~1.2.2'
-          }
+      isRuntime
+        ? undefined
+        : generatePackageJsonPlugin({
```
Comment (kagol) [inline internals/cli/src/commands/build/build-ui.ts]: 改成这样是不是好一些？
```
!isRuntime && generatePackageJsonPlugin
```

File: internals/cli/src/commands/build/build-ui.ts [hunk 2]
```diff
@@ -93,57 +93,58 @@ export const getBaseConfig = ({ vueVersion, dtsInclude, dts, buildTarget, themeV
...
-            content
-          }
-        }
-      }),
+              return {
+                filePath: filePath.replace(/[\\/]lib$/, ''),
+                content
+              }
+            }
+          }),
```
Comment (kagol) [inline internals/cli/src/commands/build/build-ui.ts]: 缩进应该是2个空格

File: internals/cli/src/commands/build/build-ui.ts (hunk 1 above)
Comment (zzcr) [reply internals/cli/src/commands/build/build-ui.ts]: 已修改

File: internals/cli/src/commands/build/build-ui.ts (hunk 2 above)
Comment (zzcr) [reply internals/cli/src/commands/build/build-ui.ts]: 已修改

**Review by kagol — APPROVED** [review]

**Pull request merged**